.azure-pipelines-templates/trace_validation.yml

```diff
@@ -10,16 +10,5 @@ steps:
       set -ex
       cd tla/
       parallel 'JSON={} ./tlc.sh consensus/Traceccfraft.tla' ::: \
-      ../build/replicate_deprecated.ndjson \
-      ../build/election_deprecated.ndjson \
-      ../build/multi_election_deprecated.ndjson \
-      ../build/check_quorum_deprecated.ndjson \
-      ../build/reconnect_deprecated.ndjson \
-      ../build/reconnect_node_deprecated.ndjson \
-      ../build/bad_network_deprecated.ndjson \
-      ../build/fancy_election.1_deprecated.ndjson \
-      ../build/fancy_election.2_deprecated.ndjson \
-      ../build/suffix_collision.1_deprecated.ndjson \
-      ../build/suffix_collision.2_deprecated.ndjson \
-      ../build/suffix_collision.3_deprecated.ndjson
+      ../build/startup.ndjson
```

Comment:
Do none of the old scenarios continue to validate?

Reply:
No, they all rely on the quirks right now, they need to be converted. I tried doing everything in one go in the other PR, but it proved difficult.